test/local/IntentJSON.test.ts

```diff
@@ -63,7 +63,7 @@ const singleSegmentIntent: IntentSegmentArgs = {
     tokenIn: '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
```

Comment:
@phanpak: @MatiasOS pointed out that the tests in this file are missing the check for the `marketSwapExactInput` segment. should we add that check in this PR?

Reply:
ah yes can do